test/powershell/Modules/Microsoft.PowerShell.Utility/Export-Alias.Tests.ps1

```diff
@@ -25,22 +34,28 @@ Describe "Export-Alias DRT Unit Tests" -Tags "CI" {
 		New-Item -Path $fulltestpath -ItemType File -Force
 		{Export-Alias $fulltestpath}| Should Not Throw
     }
-	
-	It "Export-Alias Resolve To Multiple will throw PSInvalidOperationException" {	
+
+	It "Export-Alias resolving to multiple files will throw ReadWriteMultipleFilesNotSupported" {
 		try {
-			Export-Alias *
+			$null = New-Item -Path $TestDrive\foo -ItemType File
+			$null = New-Item -Path $TestDrive\bar -ItemType File
+			Export-Alias $TestDrive\*
 			Throw "Execution OK"
-		} 
+		}
 		catch {
 			$_.FullyQualifiedErrorId | Should be "ReadWriteMultipleFilesNotSupported,Microsoft.PowerShell.Commands.ExportAliasCommand"
 		}
+		finally{
+			Remove-Item $TestDrive\foo -Force -ErrorAction SilentlyContinue
+			Remove-Item $TestDrive\bar -Force -ErrorAction SilentlyContinue

```

Comment:
$TestDrive is cleaned up only when the Describe or Context block is finished. These files are cleaned up in finally so that they do not interfere with other tests in subsequent It blocks.